backend/codesirius/problems/views/problem.py

```
@@ -1,8 +1,9 @@
 import logging
+from typing import Dict
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)